### PR TITLE
qstopmotion: 2.4.0 -> 2.4.1

### PR DESCRIPTION
**Pull request by leenaars**
###### Motivation for this change

Simple minor version bump.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/video/qstopmotion/default.nix [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
 stdenv.mkDerivation rec {
   pname = "qstopmotion";
-  version = "2.4.0";
+  version = "2.4.1";
   name = "${pname}-${version}";
 
   src = fetchurl {
-    url = "mirror://sourceforge/project/${pname}/Version_2_4_0/${name}-Source.tar.gz";
-    sha256 = "0pbyq6nrr9g3crlsng660768167s0fybvcpzbfc0w9kkhs2jwrr2";
+    url = "mirror://sourceforge/project/${pname}/Version_2_4_1/${name}-Source.tar.gz";
```
Comment (ryantm) [inline pkgs/applications/video/qstopmotion/default.nix]:
```suggestion
    url = "mirror://sourceforge/project/${pname}/Version_${builtins.replaceStrings ["."] ["_"] version}/${name}-Source.tar.gz";
```
One less thing to update each time; might also enable for automatic updates.

**Review by ryantm — CHANGES_REQUESTED** [review]

**Comment by ryantm** [comment]
@grahamcofborg build qstopmotion

**Comment by ryantm** [comment]
I'm not sure how to test this. I tried running the binary and I get this error message:
```
./results/qstopmotion/bin/qstopmotion
qt.qpa.plugin: Could not find the Qt platform plugin "xcb" in ""
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Aborted
```

**Comment by leenaars** [comment]
I think this is https://github.com/NixOS/nixpkgs/issues/37864 - conflicting Qt versions in profile...

**Review by ryantm — APPROVED** [review]

**Pull request merged**